src/controllers/dynakube/dynatraceclient/builder.go

```diff
@@ -119,7 +119,7 @@ func (dynatraceClientBuilder builder) BuildWithTokenVerification(dynaKubeStatus
 func (dynatraceClientBuilder builder) verifyTokenScopes(dynatraceClient dtclient.Client, dynaKubeStatus *dynatracev1beta1.DynaKubeStatus) error {
 	var err error
 
-	if dynatraceClientBuilder.dynakube.IsTokenScopeVerificationAllowed(timeprovider.New()) {
+	if dynatraceClientBuilder.dynakube.IsTokenScopeVerificationAllowed(timeprovider.New().SetToNow()) {
```

Comment:
And also here IMHO no timestamp freeze is needed.

Reply:
`SetTo` removed